spec/context/dimension.fmf

```diff
@@ -0,0 +1,50 @@
+summary: Supported context dimensions
+
+story:
+    As a tester I want to clearly specify dimensions, such as
+    'distro' or 'trigger', so that I can well describe the
+    context in which a test is running.
```

Comment:
should we mention all dimensions are optional, i.e. you can specify any you want?

Reply:
should we mention that the context is usually added from command line?

Reply:
should the context by automatically added by tmt run?

Reply:
Probably we should.

Reply:
I've updated the main section with more details which address above. Auto detection does not work yet, so let's add it once we have it and know how exactly it will work.